manager.spec.yaml

```diff
@@ -746,6 +746,31 @@ paths:
         - $ref: '#/components/parameters/ansible'
         - $ref: '#/components/parameters/mssql'
 
+  /cves_without_errata:
```

Comment:
I think it'd be good idea to put it under something like

```
/feature/cves_without_errata
/ff/cves_without_errata
```

etc. just to not have it on top level